rootfs/templates/wrapper

```diff
@@ -6,6 +6,9 @@ set -o pipefail
 # Geodesic Settings
 export GEODESIC_PORT=${GEODESIC_PORT:-$((30000 + $$ % 30000))}
 
+# If this env var is unset, set it to the current directory without the home dir
+export GEODESIC_WORKDIR=${GEODESIC_WORKDIR:-/localhost/$(echo "$PWD/${1#./}" | sed -e "s,^$HOME/,,g")}
```

Comment:
lets add a to `rootfs/etc/init.d/atmos.sh` which contains atmos specific enhancements.

Then let's set `export ATMOS_BASE_PATH=${GEODESIC_WORKDIR}`

Reply:
maybe we should do something like
```
if [ -d "${GEODESIC_WORKDIR}/stacks" ] && [ -d "${GEODESIC_WORKDIR/components"] ; then
  export ATMOS_BASE_PATH=${GEODESIC_WORKDIR}
fi
```

Reply:
this will still not allow running `atmos` commands from any folder in the container, but should be a good start

Reply:
> this will still not allow running atmos commands from any folder in the container, but should be a good start

This should work when the container was _started_ in a atmos root in the local filesystem.